src/include/pegasus/client.h

```diff
@@ -251,14 +251,16 @@ class pegasus_client
         filter_type sort_key_filter_type;
         std::string sort_key_filter_pattern;
         bool no_value; // only fetch hash_key and sort_key, but not fetch value
+        bool return_expire_ts;
         scan_options()
             : timeout_ms(5000),
               batch_size(100),
               start_inclusive(true),
               stop_inclusive(false),
               hash_key_filter_type(FT_NO_FILTER),
               sort_key_filter_type(FT_NO_FILTER),
-              no_value(false)
+              no_value(false),
+              return_expire_ts(true)
```

Comment:
I suggest that the default value of `return_expire_ts` should be `false`, copy data is only a case of scan, for most scan operations, expire timestamp is not a significant option.

Reply:
Yes, I'll follow this.